### DISCUSSION TITLE
Happy Women’s History Month 🧕  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#112479] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

Women’s History Month is here! As part of the festivities we’ll be highlighting organizations that get women more involved in tech and projects that are improving lives through tech where you can improve your GitHub skills. 

But no check-in would complete, if we didn’t start with some praise for our excellent community members. 

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @koeppel: Shared their expertise so a fellow member could set up an account for their new company. [They took the time to explain what solutions on GitHub could fit the company’s needs - topped off with some documentation](https://github.com/orgs/community/discussions/110865#discussioncomment-8677164). Excellent! 
* @amankr1469: Loved their [straightforward response](https://github.com/orgs/community/discussions/111371#discussioncomment-8727393) of how to add and authenticate a personal access token. 
* @ustireman: Asked a [cool question about how to break up their commits into different pull requests](https://github.com/orgs/community/discussions/111986). They’re still in need of assistance - if you can help go be someone’s hero!  

### 💟 Resources for Women in Tech: Organizations and programs to get more women involved in tech! 
* 🐍 **Pyladies**: PyLadies is a global mentorship group focused on helping more marginalized genders become active participants & leads in the Python open-source community. [Learn about joining](https://github.com/pyladies/pyladies). Our very own @ossanna16 is a member of her local chapter! 
* **Ada Developers Academy**: Ada’s mission is to prepare women and gender expansive adults to be software developers while advocating for inclusive and equitable work environments. Their program includes a collaborative learning environment, individualized support, and an applied learning internship.
* **Technovation Girls**: This program equips young women (ages 8-18) to become tech entrepreneurs and leaders. With the support of volunteer mentors and parents, girls work in teams to code mobile apps that address real-world problems. [Learn more](https://technovationchallenge.org/). 

### 🦸🏻‍♀️ Celebrate Women’s History Month: Level up your GitHub game and get involved with projects who are using technology to make the world a better place. 
* **Postpartum Wellness App**: This project is an app that helps new parents monitor their well-being during the post-partum stage. [Get involved with the project](https://github.com/BekahHW/postpartum-wellness-app). 
* **Women’s Safety Box**: This is an Android App for Women or any person, which Will Call to an Emergency number, based on a specific voice command like "Help" . It will also send SMS with current GPS coordinates to those numbers. [View the project](https://github.com/ShakilAhmedShaj/Women_Safety_Box). 
* **Tattle Made**: A community of technologists, researchers, and artists working towards a healthier online information ecosystem - visit [their repo](https://github.com/tattle-made) to learn how you can help. 

Have an open-source project that is in the spirit of Women’s History month - share it below!

**Comment by amankr1469** [comment]
Thank you @queenofcorgis ! Wish the answer got marked haha.

**Reply by queenofcorgis** [reply]
@amankr1469 😃 let me send them a quick reminder!

**Comment by queenofcorgis** [comment]
Read the latest and greatest [New to GitHub check-in](https://github.com/orgs/community/discussions/114621)!

**Comment by 123456789puscifer** [comment]
Please someone be kind enough and put me on the right path of basic coding for beginners, and what would be the must to know (what to read/look to educate myself) .. 
I know that I sound lame but I do believe with right person to guide me and pointing me where to go next and next after that,, with time I would catch up because i was always interested in softwares and comps in general,
Greetings from simple Bosnian guy 🖖👽🪬 
Peace, love & understanding for ya all.
__

**Reply by queenofcorgis** [reply]
Hi @123456789puscifer ,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
